src/main/configurations/Translate/Common/xsl/BuildError.xsl

```diff
@@ -2,15 +2,23 @@
     <xsl:output method="xml" version="1.0" encoding="UTF-8" indent="yes" omit-xml-declaration="yes"/>
     <xsl:param name="code" select="'TechnicalError'" as="xs:string" />
     <xsl:param name="reason" select="'Undefined error'" as="xs:string" />
+    <xsl:param name="cause" as="node()?" />
     <xsl:param name="details" select="''" as="xs:string" />
     <xsl:param name="detailsXml" as="node()?" />
     
     <xsl:template match="/">
         <error>
-            <code><xsl:value-of select="$code" /></code>
-            <reason><xsl:value-of select="$reason" /></reason>
-            <details><xsl:value-of select="$details" /></details>
-            <detailsXml><xsl:copy-of select="$detailsXml"/></detailsXml>
+            <xsl:choose>
+                <xsl:when test="//error">
```

Comment:
//error needs to become /error. Otherwise all already nested error structures will also be copied. We are only interested in error at root